docs/src/main/paradox/handling-blocking-operations-in-pekko-http-routes.md

```diff
@@ -66,7 +66,6 @@ For example, the above screenshot shows an Apache Pekko FJP dispatchers threads,
 named "`default-akka.default-dispatcher2,3,4`" going into the blocking state, after having been idle. 
 It can be observed that the number of new threads increases, "`default-akka.actor.default-dispatcher 18,19,20,...`" 
 however they go to sleep state immediately, thus wasting the resources.
-@java[The same happens to the global @apidoc[ForkJoinPool] when using Java Futures.]
```

Comment:
Ah damn, forgot to mention this. So basically I cannot get this to compile because even if I do a FQCN (i.e. `java.util.concurrent.ForkJoinPool`) but I noticed that both for current Akka and Pekko before this PR change it doesn't even render on the website so I just removed it.

Reply:
maybe, just add a Github issue to revisit this?

Reply:
Sure, can do

Reply:
In case it is not clear, the `@java` block would only show if `Java` is selected in the nav bar.

Reply:
But, indeed, no need to fix it now.

Reply:
> In case it is not clear, the `@java` block would only show if `Java` is selected in the nav bar.

Ah I see, but yes I will make an issue on this